test/Image1.rb

```diff
@@ -436,7 +436,7 @@ def test_clut_channel
     clut = Magick::Image.new(20, 1) { self.background_color = 'red' }
     res = nil
     expect { res = img.clut_channel(clut) }.not_to raise_error
-    assert_same(res, img)
+    expect(img).to be(res)
```

Comment:
Is there a particular order this should be? Wonder if this should be changed to `xpect(res).to be(img)`. There might be other spots where this should be changed then.

Reply:
Yeah, the usual way is `expect(actual).to be(expected)`, so this looks like they should be swapped.